client/layout/logged-out.jsx

```diff
@@ -11,32 +11,40 @@ import { connect } from 'react-redux';
  */
 import MasterbarLoggedOut from 'layout/masterbar/logged-out';
 import { getSection } from 'state/ui/selectors';
-import OauthClientLayout from 'layout/oauth-client';
-import { showOAuth2Layout } from 'state/login/oauth2/selectors';
+import OauthClientMasterbar from 'layout/masterbar/oauth-client';
+import { getOAuth2ClientData, showOAuth2Layout } from 'state/login/oauth2/selectors';
 
 const LayoutLoggedOut = ( {
+	oauth2ClientData,
 	primary,
 	section,
 	redirectUri,
 	useOAuth2Layout,
 } ) => {
-	const classes = classNames( 'layout', {
+	const classNameObject = {
 		[ 'is-group-' + section.group ]: !! section,
 		[ 'is-section-' + section.name ]: !! section,
 		'focus-content': true,
 		'has-no-sidebar': true, // Logged-out never has a sidebar
 		'wp-singletree-layout': !! primary,
-	} );
+	};
+
+	let masterbar = <MasterbarLoggedOut title={ section.title } sectionName={ section.name } redirectUri={ redirectUri } />;
```

Comment:
I don't think we should render a component we're not going to use in the case of OAuth login